<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCProductSettingFrame</class>
 <widget class="QFrame" name="ICHCProductSettingFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="alarmTimesEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Alarm Times</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Times</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="tryProductEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="productLineEdit"/>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Product</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>TryProduct</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="productClearButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Product Clear</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Wait Mold Opened Limit Time</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Sampling Interval</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="waitTimeEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="samplingEdit"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_17">
     <property name="styleSheet">
      <string notr="true">height:27px ;</string>
     </property>
     <property name="text">
      <string>Fixture</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="positiveCheckBox">
       <property name="styleSheet">
        <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
       </property>
       <property name="text">
        <string>Positive Phase</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="reversedCheckBox">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
       </property>
       <property name="text">
        <string>Reversed Phase</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Get Fail</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ICComboBox" name="getFailWay">
     <item>
      <property name="text">
       <string>Alarm When Up</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Alarm Once</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_18">
     <property name="styleSheet">
      <string notr="true">height:27px ;</string>
     </property>
     <property name="text">
      <string>Count Ways</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="ICComboBox" name="countUnitBox">
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Good</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stack-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stack-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stack-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stack-4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Bad Product</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="samplingEdit_3"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
  <customwidget>
   <class>ICComboBox</class>
   <extends>QComboBox</extends>
   <header>iccombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
